Sources/Testing/Support/Additions/TypeAdditions.swift

```diff
@@ -67,3 +67,22 @@ func isImportedFromC(_ type: Any.Type) -> Bool {
   let endIndex = mangledTypeName.index(mangledTypeName.startIndex, offsetBy: 2)
   return mangledTypeName[..<endIndex] == "So"
 }
+
+/// Check if a class is a subclass (or equal to) another class.
+///
+/// - Parameters:
+///   - subclass: The (possible) subclass to check.
+///   - superclass The (possible) superclass to check.
+///
+/// - Returns: Whether `subclass` is a subclass of, or is equal to,
+///   `superclass`.
+func isClass(_ subclass: AnyClass, subclassOf superclass: AnyClass) -> Bool {
```

Comment:
Feels like it'd read much more nicely as an extension on `AnyClass` like `func isSubclass(of superclass: Self) -> Bool`

Reply:
Would be nice. :)